data/json/recipes/recipe_others.json

```diff
@@ -3437,11 +3437,18 @@
         [ "plastic_straw", 3 ],
         [ "plastic_knife", 2 ],
         [ "plastic_spoon", 2 ],
+        [ "plastic_spoon_kids", 2 ],
```

Comment:
kids items are half the size and thus should require twice the amount.

Reply:
Didn't notice they had the "proportional" tag.